src/components/accounts/LinkdropLanding.js

```diff
@@ -70,9 +70,10 @@ class LinkdropLanding extends Component {
     }
 
     componentDidMount() {
-        const { fundingContract, fundingKey } = this.props
+        const { fundingContract, fundingKey, handleRefreshUrl } = this.props
         if (fundingContract && fundingKey) {
             this.handleCheckNearDropBalance()
+            handleRefreshUrl()
```

Comment:
force refresh url here. otherwise the `url` object is empty when reidrected back from `recover-seed-phrase` page

Reply:
@marcinbodnar @Patrick1904 Do you know what the `REFRESH_URL` action exists for?

I don't see any explicit handlers for it, so I'm guessing this is something related to our global `status` reducer maybe?

Reply:
The `handleRefreshUrl` action calls `refreshUrl` (`REFRESH_URL`) to populate the `url` object on the `account`.

Reply:
I see. Is it OK to call `handleRefreshUrl()` here?

Reply:
Yes I don't see any issue with calling it on this page when the component mounts